motr/m0crate/crate_index.c

```diff
@@ -993,6 +996,15 @@ static int cr_execute_query(struct m0_fid *id,
 
 	m0_idx_init(&idx, crate_uber_realm(), (struct m0_uint128 *) id);
 
+	if (conf->is_enf_meta && (m0_fid_is_valid(&dix_pool_ver))) {
+		idx.in_entity.en_flags |= M0_ENF_META;
+		idx.in_attr.idx_layout_type = DIX_LTYPE_DESCR;
+		idx.in_attr.idx_pver = dix_pool_ver;
+	}	
+
+	crlog(CLL_DEBUG, "DIX pool version: "FID_F"",
```

Comment:
Should we move it under condition `if (conf->is_enf_meta && (m0_fid_is_valid(&dix_pool_ver))) {` above?

Reply:
Done.